models/database_operations.py

```diff
@@ -2,26 +2,27 @@
 from models import database
 from models.database import (Admins, RobotProfile, User, UserCriteria,
                              UserRobot, engine, sessionmaker)
-
+from sqlalchemy import exists
 Session = sessionmaker(bind=engine)
 
-
 def user_in_database(password, username):
     session = Session()

```

Comment:
I'm wondering why you initialize a session every time you retrieve any data from the database 🤔. Shouldn't sqlalchemy handle it for you?

Reply:
Ok, I checked it and you can use methos from models
```python
User.query(...)
```

```python
db.session.add(item)   # db is the SQLAlchemy instance, that should be initialized inside of app.__init__
```